File: HW/HW1/HW1.py
# This is the HW1 solution done in Python

def palindrome(s):
    convert_s = ''.join(char.lower() for char in s if char.isalnum())
    return convert_s == convert_s[::-1]

s = "tacocat o tacocat"
print(palindrome(s))

File: HW/HW1/HW1.java
// This is the HW1 problem done in java. Includes an example, so should be good to compile. Feel free to pull an issue request if you find anything

public class Palindrome {
    public static boolean isPalindrome(String s) {
        String convertS = s.replaceAll("[^a-zA-Z0-9]", "").toLowerCase();
        return convertS.equals(new StringBuilder(convertS).reverse().toString());
    }

    public static void main(String[] args) {
        String s = "racecar o racecar";
        System.out.println(isPalindrome(s)); 
    }
}
